recipe/meta.yaml

```diff
@@ -0,0 +1,69 @@
+{% set name = "pypdfium2" %}
+{% set version = "4.19.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 1ca3a2ed080c263229af3fbff35ad7f751361861f10893d9908d4d852fe6eb28
+
+build:
+  skip: True  # [linux and (ppc64le or s390x)]
+  entry_points:
+    - pypdfium2 = pypdfium2.__main__:cli_main
+  script_env:
+    - PDFIUM_PLATFORM=none
+  script:
+    - cp bindings/raw.py src/pypdfium2/raw.py
+    - {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - python
+    - setuptools
+    - pip
+    - wheel !=0.38.0,!=0.38.1
+    - ctypesgen-pypdfium2-team 1.1.1+17_g10b2b4
+    - pdfium-binaries 118.0.5868
+  run:
+    - python
+    - pdfium-binaries 118.0.5868
+    # to run pypdfium2 imgtopdf you need to have pillow
+    - pillow
```

Comment:
As mentioned in https://github.com/AnacondaRecipes/pypdfium2-feedstock/pull/2#issuecomment-1698164946, pillow is supposed to be optional, only a subset of features depends on it.
i.e. it's not strictly necessary for all use cases. But it may still be OK to keep since it's very commonly needed (upstream, we had repeated questions why pillow is not in the pyproject dependencies).